cmd/call-endpoint/main.go

```diff
@@ -16,14 +17,15 @@ var args struct {
 	config.ConfigArgs
 	cmd.CommonOptions
 
-	Endpoint   string   `help:"Endpoint of the desired API." arg:""`
+	Endpoint   string   `help:"Endpoint of the desired API." arg:"" optional:""`
 	Parameters []string `help:"Parameter for the endpoint in the format 'key:value', e.g., 'id:123'." arg:"" optional:""`
 	Table      bool     `help:"Attempt to output data as a TSV. Fallback to JSON if the table conversion fails." default:"false"`
+	List       bool     `help:"List all API endpoints." default:"false"`
 }
 
 func main() {
 	kong.Parse(&args,
-		kong.Description("Execute an API request to the specified endpoint. For more information on available API endpoints, see the API resource file at: 'resources/api.json'."),
+		kong.Description("Execute an API request to a specified endpoint."),
```

Comment:
I'm not a fan of this description.
"Executing" "to" something sounds strange.
"Sending" a request "to" and endpoint makes sense.
Or maybe use "call" to be symmetric with the CMD's name.